iot-hub/Samples/service/DigitalTwinClientSamples/Thermostat/Program.cs

```diff
@@ -48,11 +48,11 @@ public static async Task Main(string[] args)
 
         private static ILogger InitializeConsoleDebugLogger()
         {
-            ILoggerFactory loggerFactory = LoggerFactory.Create(builder =>
+            using ILoggerFactory loggerFactory = LoggerFactory.Create(builder =>
             {
                 builder
                 .AddFilter(level => level >= LogLevel.Debug)
-                .AddConsole(options =>
+                .AddSimpleConsole(options =>
```

Comment:
q - what's the difference between the simple console and regular console?

Reply:
A regular console seems to be an instance of `ConosleLogger` which throws a warning regarding the use of `TimestampFormat` being deprecated. Changing it to a simple console seems to change the object type to `ConsoleFormatter` which doesn't have a deprecation warning.